lib/AST/ASTPrinter.cpp

```diff
@@ -5408,9 +5604,12 @@ void PrintAST::visitOtherConstructorDeclRefExpr(OtherConstructorDeclRefExpr *exp
 }
 
 void PrintAST::visitRebindSelfInConstructorExpr(RebindSelfInConstructorExpr *expr) {
+  Printer << expr->getSelf()->getName();
+  visit(expr->getSubExpr());
 }
 
 void PrintAST::visitMakeTemporarilyEscapableExpr(MakeTemporarilyEscapableExpr *expr) {
+  visit(expr->getSubExpr());
```

Comment:
You'll need to reconstruct the original call here, e.g:

```swift
func foo(_ fn: () -> Void) {
  withoutActuallyEscaping(fn) { fn in
    // ...
  }
}
```

Reply:
Oh looks like we actually keep track of the original expr, you should be able to print `expr->getOriginalExpr()` here.